onyxia-api/src/main/java/fr/insee/onyxia/api/security/KeycloakUserProvider.java

```diff
@@ -49,6 +49,8 @@ public UserProvider getUserProvider() {
                     .setIdep(token.getPreferredUsername())
                     .setIp(httpRequestUtils.getClientIpAddressIfServletRequestExist(((ServletRequestAttributes) RequestContextHolder.currentRequestAttributes()).getRequest()))
                     .build();
+            user.getAttributes().putAll(token.getOtherClaims());
+            user.getAttributes().put("sub",token.getSubject());
```

Comment:
I think there is no "preferred_username" in this map which is broken for standard installation of onyxia